cttportfolio.css

```diff
@@ -0,0 +1,165 @@
+* {
+	box-sizing: border-box;
+}
+html{
+	font-size:62.5%;
+}
+body{
+	margin:0;
+}
+main > div{
+	margin:0 auto;
+	max-width:1350px;
+	padding:0 1%;
+}
+/*======================================== NAV SECTION ========================================*/
+nav{
+	background-color:#ff00ff;
+	display:flex;
+	justify-content:space-around;
+	align-items:center;
+	position:sticky;
+	top:0;
+	width:100vw;
+	height:50px;
+}
+nav > a > img{
+	height:42px; 
+}
+/*======================================== HEADER SECTION ========================================*/
+header{
+	background-color:#808080;
+	width:100vw;
+	height:calc(100vh - 50px);
+	display:flex;
+	align-items:center;
+	justify-content:center;
+}
+/*======================================== ABOUT ME SECTION ========================================*/
+.aboutmediv{
+	background-color:#1f7a1f;
+}
+.aboutmemain{
+	display:flex;
+	flex-direction:column;
+	align-items:center;
+}
+.aboutmemainchild{
+	background-color:#33cc33;
+	display:flex;
+	width:100%;
+}
+.aboutmeimg{
+	width:25%;
+	display:flex;
+	justify-content:center;
+}
+
+.aboutmeimg>img{ /*634x951 this is teh stock image */
+	max-width:634px;
+	max-height:951px;
+	width:inherit;
+	height:auto;
+}
+.aboutmetext{
+	padding-right:15%;
+}
+.aboutmesub{
+	background-color:#70db70;
+}
+/*======================================== PORTFOLIO SECTION ========================================*/
+.portfolio{
+	background-color:#0040ff;
+}		
+.topportfolio{
+	display:flex;
+	flex-direction:column;
+	align-items:center;
+}
+.mainportfolio{
+	background-color:#1a75ff;
+	display:flex;
+	flex-wrap:wrap;
+	width:100%;
+}
+.mainportfolio > a{
+	width:100%;
+}
+.mainprojectlink{
+	display:flex;
+}
+.mainprojectlink > *{
+	display:block;
+	width:50%;
+	height:50%;
+
+}
+.subportfolio{
+	background-color:#00bfff;
+	display:flex;
+	justify-content:space-between;
+}
+.subportfolio > a{
+	width:33%;
+	height:33%;
+}
+.subprojectlink > *{
+	width:100%;
+	height:100%;
+	display:flex;
+	flex-direction:column;
+	align-items:center;
+}
+.mainprojectlink > img{
+	object-fit:cover;
+}
+a.portfoliolinks:hover{
+	border:1px solid black;	
+}
+
+/*======================================== CONTACT SECTION ========================================*/
+.contactme{
+	background-color:red;
+}
+.contactmetext{
+	display:flex;
+	justify-content:space-between;
+	padding:0 10%;
+}
+/*======================================== FOOTER SECTION ========================================*/
+footer{
+	background-color:#ffff66;
+	display:flex;
+	justify-content:space-between;
+	padding:0 30%;
+}
+/*======================================== PROJECT PAGE SECTION ========================================*/
+.projectpageinfo{
+	display:flex;
+	justify-content:space-between;
+	width:1350px;
+	margin:0 auto;
+	height:calc(80vh - 50px);
+}
+.projectpageinfo > *{
```

Comment:
Not sure what you're trying to achieve with the `> *`?

Reply:
I am using it to target all child elements of the class